var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -1,103 +1,112 @@
 
-    version('develop', git=git, branch='master', clean=False)
-    version('2.8.0', git=git, tag='2.8.0', clean=False)
-    version('2.7.3', git=git, tag='2.7.3', clean=False)
-    version('2.7.2', git=git, tag='2.7.2', clean=False)
-    version('2.7.0', git=git, tag='2.7.0', clean=False)
-    version('2.6.0', git=git, tag='2.6.0', clean=False)
-    version('2.5.0', git=git, tag='2.5.0', clean=False)
-    version('2.4.3', git=git, tag='2.4.3', clean=False)
-    version('2.4.1', git=git, tag='2.4.1', clean=False)
-    version('2.3.4', git=git, tag='2.3.4', clean=False)
-    version('2.3.3', git=git, tag='2.3.3', clean=False)
-    version('2.2.1', git=git, tag='2.2.1', clean=False)
-
-    variant('python', default=False, description="Enable Python Neurodamus")
-    variant('common', default=False, description="Merge in synapse mechanisms hoc & mods")
-
-    # Attempt to support building
-    depends_on('neuron~binary+python', when='+common')
-
-    # Neurodamus py is currently an extension to core
-    resource(name='pydamus',
-             git='ssh://bbpcode.epfl.ch/sim/neurodamus-py',
-             when='+python',
-             destination='resources')
+    version('develop', branch='master', clean=False)
+    version('2.9.1', tag='2.9.1', clean=False)
+    version('2.9.0', tag='2.9.0', clean=False)
+    version('2.8.0', tag='2.8.0', clean=False)
+    version('2.7.3', tag='2.7.3', clean=False)
+    version('2.7.2', tag='2.7.2', clean=False)
+    version('2.7.0', tag='2.7.0', clean=False)
+    version('2.6.0', tag='2.6.0', clean=False)
+    version('2.5.0', tag='2.5.0', clean=False)
+    version('2.4.3', tag='2.4.3', clean=False)
+    version('2.4.1', tag='2.4.1', clean=False)
+    version('2.3.4', tag='2.3.4', clean=False)
+    version('2.3.3', tag='2.3.3', clean=False)
+    version('2.2.1', tag='2.2.1', clean=False)
+
+    variant('mpi',    default=True,  description="Enable MPI support")
+    variant('common', default=False, description="Bring in common synapse mechanisms")
+    variant('hdf5',   default=True,  description="Enable old Hdf5 reader")
+    variant('reportinglib', default=True, description="Enable ReportingLib")
+    variant('synapsetool',  default=True, description="Enable SynapseTool reader (for edges)")
+    variant('mvdtool',      default=True, description="Enable MVDTool reader (for nodes)")
+
+    # NOTE: Several variants / dependencies come from SimModel
+    depends_on("mpi",  when='+mpi', type=('build', 'run'))  # dont link
+    depends_on("hdf5+mpi", when='+hdf5+mpi')
+    depends_on("hdf5~mpi", when='+hdf5~mpi')
+    depends_on('reportinglib',         when='+reportinglib')
+    depends_on('reportinglib+profile', when='+reportinglib+profile')
+    depends_on('synapsetool',          when='+synapsetool')
+    depends_on('py-mvdtool',           when='+mvdtool', type='run')
+    # If external & static, we must bring their dependencies.
+    depends_on('zlib')  # for hdf5
```

Comment:
nitpick : I see that zlib is dependency of hdf5 but wondering in what context you saw link error.

Reply:
When hdf5 is external and static. Hence the comment

Reply:
In that case ‘libs’ property of hdf5 should provide necessary libraries? 
Hmm...but I see the possible issues there...it’s ok as it is.